db/migrate/20230322213633_remove_emoji_prefix_from_talks.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveEmojiPrefixFromTalks < ActiveRecord::Migration[7.0]
+  def up
+    Talk.all.each { _1.update!(title: _1.title.delete_prefix('💬').delete_prefix('⚡'.strip)) }
```

Comment:
Le strip est au mauvais endroit ici: 

```suggestion
    Talk.all.each do |talk|
      title = talk.title.delete_prefix('💬').delete_prefix('⚡').strip
      talk.update!(title: title)
    end
```